tests/framework/pkg/session/session.go

```diff
@@ -34,13 +39,13 @@ func (ts *Session) RegisterCleanupFunc(f CleanupFunc) {
 // Cleanup this method will call all registered cleanup functions in order and close the test session.
 func (ts *Session) Cleanup() {
 	ts.open = false
-
 	for i := len(ts.cleanupQueue) - 1; i >= 0; i-- {
 		err := ts.cleanupQueue[i]()
 		if err != nil {
 			ts.testingT.Logf("error calling cleanup function: %v", err)
 		}
 	}
+	ts.cleanupQueue = []CleanupFunc{}
```

Comment:
don't clear the queue we should be able to call cleanup multiple times because cleanup functions should be idempotent